mappings/net/minecraft/nbt/NbtOps.mapping

```diff
@@ -58,17 +69,89 @@ CLASS net/minecraft/unmapped/C_fybaywjo net/minecraft/nbt/NbtOps
 		ARG 1 element
 	METHOD getStringValue (Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
 		ARG 1 element
+	METHOD m_kvzwcnej tryUnwrap (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_oivssbvb;
+		ARG 0 nbt
+	METHOD m_vpbkfzmz createCollector (Lnet/minecraft/unmapped/C_oivssbvb;)Ljava/util/Optional;
+		ARG 0 element
 	METHOD mapBuilder mapBuilder ()Lcom/mojang/serialization/RecordBuilder;
 	METHOD mergeToList (Ljava/lang/Object;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
 		ARG 1 list
 		ARG 2 value
+	METHOD mergeToList (Ljava/lang/Object;Ljava/util/List;)Lcom/mojang/serialization/DataResult;
+		ARG 1 list
+		ARG 2 values
 	METHOD mergeToMap (Ljava/lang/Object;Lcom/mojang/serialization/MapLike;)Lcom/mojang/serialization/DataResult;
 		ARG 1 element
+		ARG 2 values
 	METHOD mergeToMap (Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
 		ARG 1 map
 		ARG 2 key
 		ARG 3 value
+	METHOD mergeToMap (Ljava/lang/Object;Ljava/util/Map;)Lcom/mojang/serialization/DataResult;
+		ARG 1 element
+		ARG 2 values
 	METHOD remove (Ljava/lang/Object;Ljava/lang/String;)Ljava/lang/Object;
 		ARG 1 element
 		ARG 2 key
+	CLASS C_cfjsyyyh SimpleListCollector
+		FIELD f_zdfsryne result Lnet/minecraft/unmapped/C_hqhtfwlr;
+		METHOD <init> (Lnet/minecraft/unmapped/C_hqhtfwlr;)V
+			ARG 1 nbt
+		METHOD <init> (Lnet/minecraft/unmapped/C_oivssbvb;)V
+			ARG 1 element
 	CLASS C_cmafvjrt MapBuilder
+		METHOD append (Ljava/lang/String;Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;
+			ARG 1 key
+			ARG 2 element
+			ARG 3 nbt
+		METHOD build (Ljava/lang/Object;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+			ARG 1 nbt
+			ARG 2 element
+	CLASS C_cpsqmvih ByteListCollector
+		FIELD f_ijtscyyp values Lit/unimi/dsi/fastutil/bytes/ByteArrayList;
+		METHOD <init> (B)V
+			ARG 1 value
+		METHOD <init> ([B)V
+			ARG 1 values
+	CLASS C_dwkvaauw LongListCollector
+		FIELD f_rjyiwlrj values Lit/unimi/dsi/fastutil/longs/LongArrayList;
+		METHOD <init> (J)V
+			ARG 1 value
+		METHOD <init> ([J)V
+			ARG 1 values
+	CLASS C_lrggpudd DelegatingListCollector
+		FIELD f_oknzgzfq INSTANCE Lnet/minecraft/unmapped/C_fybaywjo$C_lrggpudd;
+	CLASS C_nehsnwyp ListCollector
+		METHOD m_berihwml accept (Lnet/minecraft/unmapped/C_oivssbvb;)Lnet/minecraft/unmapped/C_fybaywjo$C_nehsnwyp;
+			ARG 1 element
+		METHOD m_fjtdnzmx accept (Ljava/lang/Iterable;)Lnet/minecraft/unmapped/C_fybaywjo$C_nehsnwyp;
+			ARG 1 elements
+		METHOD m_gxpktbdp acceptAll (Ljava/util/stream/Stream;)Lnet/minecraft/unmapped/C_fybaywjo$C_nehsnwyp;
```

Comment:
```suggestion
		METHOD m_gxpktbdp accept (Ljava/util/stream/Stream;)Lnet/minecraft/unmapped/C_fybaywjo$C_nehsnwyp;
			ARG 1 elements
```
or the `Iterable` one should also be `acceptAll`, but they should match

Reply:
i prefer this, good catch